src/sonic-host-services/scripts/check_readonly_disk

```diff
@@ -0,0 +1,16 @@
+#! /bin/bash
+
+echo "checking for read-only disk"
+SYSTEM_ERR=4
+
+touch /var/log/syslog
+ret=$?
+
+echo "ret=${ret}"
+
+if [ ${ret} -eq 0 ]; then
+    exit 0
+else
+    echo "Read-Only system"
```

Comment:
logger

Reply:
The echo will show this info to the user logging in as below. Thought this might be useful.
Outside this, the monitor process could watch the disk status & log required error for alerts.

**Read-Only system**
Linux str-s6000-acs-13 4.19.0-9-2-amd64 #1 SMP Debian 4.19.118-2+deb10u1 (2020-06-07) x86_64
You are on
  ____   ___  _   _ _  ____
 / ___| / _ \| \ | (_)/ ___|
 \___ \| | | |  \| | | |
  ___) | |_| | |\  | | |___
 |____/ \___/|_| \_|_|\____|

-- Software for Open Networking in the Cloud --

Unauthorized access and/or use are prohibited.
All access and/or use are subject to monitoring.